packages/docusaurus-theme-classic/src/theme/components/Playground/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as React from 'react';
+import {LiveProvider, LiveEditor, LiveError, LivePreview} from 'react-live';
+
+function Playground({children, theme, transformCode, ...props}) {
+  return (
+    <LiveProvider
+      code={children}
+      transformCode={transformCode || (code => `${code};`)}
+      theme={theme}
+      {...props}>
+      <div
```

Comment:
Must we use all these inline styles? Can we use CSS modules? Or do we want this part to be independent of webpack config? Inline styles are hard to override and I suspect users might want to do so.

Reply:
yeah actually wanted to convert it to css modules. just havent done it yet